ait/dsn/bin/ait_sle_bridge.py

```diff
@@ -65,16 +65,16 @@ def process_pdu(raf_mngr):
 
 
 if __name__ == '__main__':
-    raf_mngr = ait.dsn.sle.RAF(hostname='atb-ocio-sspsim.jpl.nasa.gov', port=5100,
-                             auth_level="bind",
-                             inst_id="sagr=LSE-SSC.spack=Test.rsl-fg=1.raf=onlc1")
+    raf_mngr = ait.dsn.sle.RAF(hostname=self._hostname, port=self._port,
+                             auth_level=self._auth_level,
```

Comment:
This code is not in a class so `self.` isn't going to work.

Reply:
Good catch. I removed all the redundant arguments which are already present in config.yaml and inherited from the SLE base class defined in common.py (hostname, port, auth_level). The only required argument when instantiating a RAF/RCF/CLTU class is inst_id, which isn't defined in config.yaml because it's specific to the object being instantiated.